shared/constants/chat2/message.js

```diff
@@ -550,6 +550,14 @@ const uiMessageToSystemMessage = (minimum, body, reactions): ?Types.Message => {
         ...minimum,
       })
     }
+    case RPCChatTypes.localMessageSystemType.changeretention: {
```

Comment:
will have a full version of this in the next PR, but flow was unhappy with the new type missing from the switch